src/UiPath.Workflow/Activities/VbExpressionValidator.cs

```diff
@@ -74,7 +74,6 @@ protected override void UpdateCompilationUnit(ExpressionContainer expressionCont
                 mainTypeName: null,
                 globalImports: globalImports,
                 rootNamespace: "",
-                optionStrict: OptionStrict.On,
```

Comment:
This will have some downstream effects so I want to be sure it's needed. My understanding is that at runtime strict is off. At design time Studio Web depends on strict to be on and goes through some extra steps to make sure the types are correct before validation. We can change the option in the designer but it's just important to make sure that happens. I'm not sure what Studio needs for strict. Not for this PR but I'm wondering if we should add an option somewhere to control strict without having to subclass the whole validator.

Reply:
The reason I disabled it is because it was causing warnings such as this one: 

```warning BC40057: Namespace or type specified in the project-level Imports 'System.Activities.DynamicUpdate' doesn't contain any public member or cannot be found. Make sure the namespace or the type is defined and contains at least one public member. Make sure the imported element name doesn't use any aliases.	```

-- one for each expression it compiles, so you'd end up with a lot of them in any decent project.

![image](https://user-images.githubusercontent.com/11870708/210642129-8360b35a-eda1-47e2-ab01-d5c7a874e642.png)

From what I tested, this didn't happen on the old validation flow, so I removed the option in order to maintain the existing behavior. If you have any other idea as to what might be causing this instead, I could look more into it.